rapids-cmake/export/detail/post_find_package_code.cmake

```
@@ -0,0 +1,88 @@
+#=============================================================================
+# Copyright (c) 2023, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#=============================================================================
+include_guard(GLOBAL)
+
+#[=======================================================================[.rst:
+rapids_export_post_find_package_code
+------------------------------------
+
+.. versionadded:: v23.12.00
+
+Record for <PackageName> a set of CMake instructions to be executed after the package
+has ben found successfully.
+
+.. code-block:: cmake
+
+  rapids_export_post_find_package_code((BUILD|INSTALL)
+                                       <PackageName>
+                                       <code>
+                                       (<ExportSetName> | EXPORT_SET [ExportSetName])
```

Assuming the other function has both options for both primarily for backwards compat, could we just require the `EXPORT_SET` keyword in this new function? Both calls I see are using it anyway.
```suggestion
                                       EXPORT_SET [ExportSetName]
```

---

Done ( Had to revise tests as well ).